### ISSUE TITLE
I can't stash changes by command ' git stash save "test" '

### ISSUE THREAD

**Issue by yao012** [git-for-windows/git#1739]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

** 
git version 2.18.0.windows.1
cpu: x86_64
built from commit: cd1a74fc9dc8a07626c216940db9a51f25206e03
sizeof-long: 4
 **
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

** Microsoft Windows [版本 6.1.7601] **
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

** 
Editor Option: VIM
Path Option: Cmd
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Disabled
Enable Symlinks: Disabled 
**
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** nothing importance **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** CMD **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** git stash save "test" **
```
 - What did you expect to occur after running these commands?

** My local changes can be stashed **

 - What actually happened instead?

** Cannot save the current index state **

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** no,This phenomenon occurs at our company's internal address**

**Comment by PhilipOakley** [comment]
@TheFerrango , @yao012 would it be possible to check if this happens on a Linux test machine? 

Also it is worth reporting this upstream to Git Mailing List `git@vger.kernel.org` (emails MUST be plain text - all HTML is treated as spam, ahhh the good old days..)

The key point is the "only breaks if I try to name my stash", which may get a lot quicker fix. 

Though it may already be fixed in v2.19 which is due imminently. https://github.com/git/git/blob/master/Documentation/RelNotes/2.19.0.txt - but it's not mentioned in the rel notes.

**Comment by TheFerrango** [comment]
it does not happen on a linux machine.

Tested on a fresh Ubuntu 18.04 install with Git 2.17.1

**Comment by yao012** [comment]
@PhilipOakley  you are right ，it did get fixed in the new version v2.19

**Comment by PhilipOakley** [comment]
@yao012 Thanks for testing and reporting. I'll close the issue now if that's OK.
